lib/greet.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = exports = {};
+
+var strangerName = process.argv[2];
```

Comment:
Nice work storing the process.argv into a variable to use more cleanly in your code later. :)